pkg/occlient/occlient.go

```diff
@@ -254,6 +255,44 @@ func (c *Client) isLoggedIn() bool {
 	return true
 }
 
+// RunLogout logs out the current user from cluster
+func (c *Client) RunLogout() error {
+	output, err := c.userClient.Users().Get("~", metav1.GetOptions{})
+	if err != nil {
+		glog.V(1).Infof("%v : unable to get userinfo", err)
+	}
+
+	conf, err := c.KubeConfig.ClientConfig()
+	if err != nil {
+		glog.V(1).Infof("%v : unable to get client config", err)
+	}
+	client, err := oauthv1client.NewForConfig(conf)
+	if err != nil {
+		glog.V(1).Infof("%v : unable to create a new OauthV1Client", err)
+	}
+
+	if err := client.OAuthAccessTokens().Delete(conf.BearerToken, &metav1.DeleteOptions{}); err != nil {
+		glog.V(1).Infof("%v", err)
+	}
+
+	rawConfig, err := c.KubeConfig.RawConfig()
+	if err != nil {
+		glog.V(1).Infof("%v : unable to switch to  project", err)
+	}
+	for key, value := range rawConfig.AuthInfos {
+		if key == rawConfig.Contexts[rawConfig.CurrentContext].AuthInfo {
+			value.Token = ""
+		}
+	}
+	err = clientcmd.ModifyConfig(clientcmd.NewDefaultClientConfigLoadingRules(), rawConfig, true)
+	if err != nil {
+		glog.V(1).Infof("%v : unable to write config to config file", err)
+	}
+
+	fmt.Printf("Logged \"%v\" out on \"%v\"\n", output.Name, conf.Host)
```

Comment:
Please move this print statement to the cmd/logout directory

Reply:
I don't know why you thought like that, Basically this step should be done inside the function in my opinion

Reply:
occleint does occlient related actions and cmd package is used for cmd and output related stuff. This will mix the functionality of the two packages